ext/fs/in_memory_fs.rs

```diff
@@ -280,8 +281,8 @@ impl FileSystem for InMemoryFs {
     self.lstat_sync(&path)
   }
 
-  fn realpath_sync(&self, _path: &Path) -> FsResult<PathBuf> {
-    Err(FsError::NotSupported)
+  fn realpath_sync(&self, path: &Path) -> FsResult<PathBuf> {
+    RealFs.realpath_sync(path)
```

Comment:
I don't think this will work for the in memory file system because it will contain paths not on the real file system?

Reply:
Yeah I'm aware. I'm still testing some stuff across crates. I'll let you know when it's ready